my-app/config/database.php

```diff
@@ -15,7 +15,7 @@
     |
     */
 
-    'default' => env('DB_CONNECTION', 'mongodb'),
+    'default' => env('mongodb'),
```

Comment:
I: this syntax causes an error for me when I run a query using DB::collection(). I think you can just directly assign 'mongodb' to 'default':

```suggestion
    'default' => 'mongodb',
```

Reply:
Oops, thought I had deleted that part. Good catch, will fix!